sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/DataCollection/DocumentHelper.cs

```diff
@@ -19,7 +21,87 @@ internal static class DocumentHelper
     {
         // TODO: NEED TO HANDLE UNIQUE MAXLENGTH VALUES FOR DOCUMENT TYPES. SEE SWAGGER FOR MAXLENGTH VALUES.
 
-        internal static RemoteDependency ConvertToRemoteDependency(Activity activity)
+        #region Document Buffer Extension Methods
+        public static void AddExceptionDocument(this DoubleBuffer buffer, ActivityEvent activityEvent)
```

Comment:
Do you want to keep extension methods in a separate file like DoubleBufferExtensions?

Reply:
i did this originally, but decided to merged the two because i thought it wouldn't pass review. haha :)